roles/docker/tasks/main.yml

```diff
@@ -1,35 +1,17 @@
 ---
 # tasks file for docker
-- name: configure docker consul dns
+- name: deploy docker service
   sudo: yes
-  lineinfile:
-    dest: /etc/sysconfig/docker
-    regexp: ^OPTIONS=
-    line: 'OPTIONS=\"--storage-driver=overlay --graph={{ docker_graph_dir }} --dns 172.17.0.1 --dns 8.8.8.8 --dns-search service.{{ consul_domain }} \"'
-    state: present
-    create: yes
+  sudo_user: root
+  template:
+    src: docker.service.j2
+    dest: /etc/systemd/system/docker.service
   notify:
+    - reload systemd
     - restart docker
   tags:
     - docker
 
-#- name: configure docker temporary directory
-#  sudo: yes
-#  lineinfile:
-#    dest: /etc/sysconfig/docker
-#    state: present
-#    line: 'DOCKER_TMPDIR=\"{{ docker_tmp_dir }}\"'
-#  notify:
-#    - restart docker
-
-#- name: configure docker proxy
```

Comment:
this file is ignored by coreOS. This is handle now in  roles/docker/templates/docker.service.j2
We could override DOCKER_OPTS environment variable here /etc/systemd/system/docker.service.d/
but coreOS is giving a memory error message when using more than one flag there
https://coreos.com/os/docs/877.1.0/customizing-docker.html